interop/xds/client/client.go

```diff
@@ -94,7 +94,7 @@ func (s *statsService) GetClientStats(ctx context.Context, in *testpb.LoadBalanc
 		select {
 		case r := <-watcher.c:
 			if r != nil {
-				watcher.rpcsByPeer[(*r).GetServerId()]++
+				watcher.rpcsByPeer[(*r).GetHostname()]++
```

Comment:
Will we need to handle server_id for other purposes?

Reply:
I will likely need to incorporate server_id when using this client for our internal (grpclbv2) tests; on GCP, using hostname is convenient since hostname=VM instance name. Will revisit once this framework is brought in for use with the internal-only tests.